hoomd/logging.py

```diff
@@ -625,25 +628,49 @@ class they come from. For instance, the `hoomd.md.pair.LJ` class has a
         occur for user specified names that are reused.
 
     Args:
-        categories (`list` [`str`], optional): A list of string categories
-            (list of categories can be found in `LoggerCategories`). These are
-            the only types of loggable quantities that can be logged by this
-            logger. Defaults to allowing every type.
+        categories (`list` [`str` ], `LoggerCategories`, optional): Either a
+            list of string categories (list of categories can be found in
+            `LoggerCategories`) or a `LoggerCategories` instance with the
+            desired flags set. These are the only types of loggable quantities
+            that can be logged by this logger. Defaults to allowing every type
+            ``LoggerCategories.ALL``.
         only_default (`bool`, optional): Whether to log only quantities that are
             logged by default. Defaults to ``True``. Non-default quantities
             are typically measures of operation performance rather than
             information about simulation state.
 
-    .. rubric:: Example:
+    .. rubric:: Examples
+
+    There are various ways to create a logger with different available
+    loggables. To allow all loggable categories just create a logger.
 
     .. code-block:: python
 
+        from hoomd.logging import LoggerCategories
+
         logger = hoomd.logging.Logger()
+
+    When restricting to a subset of categories you can use a list of strings.
+
+    .. code-block:: python
+
+        logger = hoomd.logging.Logger(categories=["string", "strings"])
+
+    The `LoggerCategories` enum values can also be used directly.
+
+    .. code-block:: python
+
+        logger = hoomd.logging.Logger(
+            categories=LoggerCategories.particle | LoggerCategories.bond)
```

Comment:
```suggestion
    Use a list of strings to log a subset of categories:

    .. code-block:: python

        logger = hoomd.logging.Logger(categories=["string", "strings"])
```
I think using the enum directly is an advanced topic and therefore doesn't need an example. Users that need it can infer the appropriate syntax from the reference documentation.